app/services/data_migrations/fix_empty_offer_conditions.rb

```diff
@@ -0,0 +1,18 @@
+module DataMigrations
+  class FixEmptyOfferConditions
+    TIMESTAMP = 20210624092858
+    MANUAL_RUN = false
+
+    def change
+      OfferCondition.where(text: '').find_each do |oc|
+        application_choice = oc.offer.application_choice
+
+        if application_choice.status == 'pending_conditions'
```

Comment:
Couldn't one of multiple conditions be set to an empty string?

Reply:
Ah yes good point, this is too simplistic - will update - thanks!